### DISCUSSION TITLE
How to use this library with PyGame + PyOpenGL?

### DISCUSSION THREAD

**Discussion by LukovDev** [pthom/imgui_bundle#151] [Q&A]
I'm writing my own engine, and I need an interface. Writing your own interface is not an easy task, so I want to use this library

My window class is based on PyGame and PyOpenGL
This class is very large, and it is extremely impractical to redo it for PySDL2. What to do?

**Comment by pthom** [comment]
Hello,

There are full python backends here: https://github.com/pthom/imgui_bundle/tree/main/bindings/imgui_bundle/python_backends

They are inspired from [pyimgui](https://github.com/pyimgui/pyimgui) backends.

A draft for pygame is located the `python_backends_disabled_folder`, with an example inside the folder `examples_disabled`. 

At the moment, the rendering is incorrect:
<img width="405" alt="image" src="https://github.com/pthom/imgui_bundle/assets/7694091/942a3489-0b47-462b-bc2b-96b700c310b2">


This is probably because the pygame backend uses the OpenGL FixedPipelineRenderer, whereas the working glfw python backend uses a more modern ProgrammablePipelineRenderer.

I do not have time to work on this now. Do you think you would have time investigate this on your side?
